tests/integrations/config/dynamic/raftstore.rs

```diff
@@ -224,3 +224,58 @@ fn test_update_raftstore_config() {
 
     system.shutdown();
 }
+
+#[test]
+fn test_update_raftstore_io_config() {
```

Comment:
it will be better to add some write traffic to verify the io thread count decrement logic.